File: java/java-tests/testData/refactoring/inlineLocal/InLambda.java
import java.util.Collection;
import java.util.List;
import java.util.function.Predicate;
import java.util.stream.Collectors;

public final class InlineTest {
  public List<Object> foo(Collection<Object> bars, Predicate<String> filter) {
    return bars.stream().filter(bar -> {
      final String se<caret>archString = bar.toString();
      return filter.test(searchString);
    }).collect(Collectors.toList());
  }
}

File: plugins/kotlin/idea/tests/testData/exitPoints/inLambda.kt
fun foo(f: (String?) -> Int) {}

fun test() {
    <info descr="null">foo</info> {
        if (it == null) <info descr="null">return@~foo 1</info>
        (1+1)
        if (it == "a") <info descr="null">2</info> else <info descr="null">0</info>
    }
}